x/profiles/types/account.go

```diff
@@ -137,6 +150,106 @@ func (p *Profile) SetSequence(sequence uint64) error {
 	return p.setAccount(acc)
 }
 
+// LockedCoins implements exported.VestingAccount
+func (p *Profile) LockedCoins(blockTime time.Time) sdk.Coins {
+	acc := p.getVestingAccount()
+	if acc == nil {
+		return nil
```

Comment:
@RiccardoM 
Should it return sdk.Coins() like [cosmos-sdk](https://github.com/cosmos/cosmos-sdk/blob/d07e41683a3a4d16bebd3c07a65f3c9a4de0baba/x/bank/keeper/view.go#L179), instead of `nil`?

Not sure it would happen bugs with nil slice or not.
[Reference](https://medium.com/@habibridho/golang-nil-vs-empty-slice-87fd51c0a4d)

Reply:
I think we should follow the Cosmos conventions. I've changed them to be `sdk.NewCoins` instead of `nil`